internal/examples/thrift-keyvalue/keyvalue/client/main.go

```diff
@@ -56,9 +57,11 @@ func main() {
 	var outbound transport.UnaryOutbound
 	switch strings.ToLower(outboundName) {
 	case "http":
-		outbound = httpTransport.NewSingleOutbound("http://127.0.0.1:24035")
+		outbound = httpTransport.NewSingleOutbound("http://127.0.0.1:24042")
```

Comment:
I'm trying to get less overlapping ports/put more distance between them for the future. In a cleanup PR, I want to get these actually auto-allocated, but this is out of scope for here.